src/Native/CMakeLists.txt

```diff
@@ -126,18 +126,14 @@ if(CLR_CMAKE_PLATFORM_ARCH_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
```

Comment:
There is block of similar defines in src\Native\Bootstrap\CMakeLists.txt that we should be able to delete since they should get inherited from this CMakeLists.txt .